internal/constants/constants.go

```diff
@@ -19,4 +19,4 @@ package constants
 const RatifyPolicy = "ratify-policy"
 const EmptyNamespace = ""
 const NamespaceSeperator = "/"
-const MaxBriefErrLength = 30
+const MaxBriefErrLength = 150
```

Comment:
curious, is 150 the limit from my partners?

Reply:
There is not specifc request on the limit, I just picked up 150 which is recommended by @yizha1 in the issue. @yizha1 wonder if it's a reasonable limit for most error messages.